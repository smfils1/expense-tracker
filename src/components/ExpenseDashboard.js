import React from 'react';
import ExpenseList from './ExpenseList';
import ExpensesSummary from './ExpensesSummary';

import ExpenseListFilters from './ExpenseListFilters'


const ExpenseDashboard = (props) => (
    <div>
        <ExpensesSummary />
        <ExpenseListFilters/>
        <ExpenseList/>         
    </div>

)


export default ExpenseDashboard