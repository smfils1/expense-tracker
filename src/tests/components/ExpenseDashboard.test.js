import React from 'react'
import {shallow} from 'enzyme'
import ExpenseDashboard from "../../components/ExpenseDashboard"

describe('<ExpenseDashboard/>',()=>{
    test('Should render expense dashboard list correctly',()=>{
        const wrapper = shallow(<ExpenseDashboard/>)
        expect(wrapper).toMatchSnapshot()
    })
})