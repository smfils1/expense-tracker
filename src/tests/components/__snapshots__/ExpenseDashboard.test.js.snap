// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ExpenseDashboard/> Should render expense dashboard list correctly 1`] = `
<div>
  <ConnectFunction />
  <ConnectFunction />
  <ConnectFunction />
</div>
`;
